doc/barman-cloud-backup-delete.1.md

```diff
@@ -42,69 +42,70 @@ This script and Barman are administration tools for disaster recovery
-
--V, --version
-:    show program's version number and exit
-
--v, --verbose
-:    increase output verbosity (e.g., -vv is more than -v)
-
--q, --quiet
-:    decrease output verbosity (e.g., -qq is less than -q)
-
--t, --test
-:    test connectivity to the cloud destination and exit
-
--b *BACKUP_ID*, --backup-id *BACKUP_ID*
-:    a valid Backup ID for a backup in cloud storage which is to be deleted
-
--r *RETENTION_POLICY*, --retention-policy *RETENTION_POLICY*
-:    used instead of --backup-id, a retention policy for selecting the backups
-     to be deleted, e.g. "REDUNDANCY 3" or "RECOVERY WINDOW OF 2 WEEKS"
-
---dry-run
-:    run without actually deleting any objects while printing information
-     about the objects which would be deleted to stdout
-
---batch-size *SIZE*
-:    The maximum number of objects to be deleted in a single request to the cloud
-     provider. If unset then the maximum allowed batch size for the specified cloud
-     provider will be used (1000 for aws-s3, 256 for azure-blob-storage and 100 for
-     google-cloud-storage).
-
---cloud-provider {aws-s3,azure-blob-storage,google-cloud-storage}
-:    the cloud provider to which the backup should be uploaded
-
--P, --profile
-:    profile name (e.g. INI section in AWS credentials file)
-
---endpoint-url
-:    override the default S3 URL construction mechanism by specifying an endpoint.
-
---read-timeout *TIMEOUT*
-:    the time in seconds until a timeout is raised when waiting to read from a
-     connection to AWS S3 (defaults to 60 seconds)
-
---credential {azure-cli,managed-identity}
-:    optionally specify the type of credential to use when authenticating with
-     Azure Blob Storage. If omitted then the credential will be obtained from the
-     environment. If no credentials can be found in the environment then the default
-     Azure authentication flow will be used.
-
+# Usage
+```bash
+usage: barman-cloud-backup-delete [-V] [--help] [-v | -q] [-t]
+                                  [--cloud-provider {aws-s3,azure-blob-storage,google-cloud-storage}]
+                                  [--endpoint-url ENDPOINT_URL] [-P PROFILE]
+                                  [--read-timeout READ_TIMEOUT]
+                                  [--credential {azure-cli,managed-identity}]
+                                  (-b BACKUP_ID | -r RETENTION_POLICY)
+                                  [--dry-run] [--batch-size DELETE_BATCH_SIZE]
+                                  source_url server_name
+
+This script can be used to delete backups made with barman-cloud-backup
+command. Currently AWS S3, Azure Blob Storage and Google Cloud Storage are
+supported.
+
+positional arguments:
+  source_url            URL of the cloud source, such as a bucket in AWS S3.
+                        For example: `s3://bucket/path/to/folder`.
+  server_name           the name of the server as configured in Barman.
+
+optional arguments:
+  -V, --version         show program's version number and exit
```

Comment:
These apostrophes are being interpreted within the triple backticks section, leading to the rest of the usage section being rendered in a different colour.